__std__/cells/plutus/library/make-plutus-shell.nix

```diff
@@ -19,21 +19,74 @@ inputs.std.lib.dev.mkShell {
       category = "general commands";
       help = "For nix maintainers: build everything in the flake";
     }
-    {
-      package = cell.packages.fix-cabal-fmt;
-      category = "plutus";
-      help = "Format all cabal files in-place";
-    }
     {
       package = cell.packages.fix-png-optimization;
-      category = "plutus";
+      category = "general commands";
       help = "Fix all PNG files in-place";
     }
+    {
+      package = pkgs.shellcheck;
+      category = "general commands";
+      help = "Shell file checker";
+    }
+    {
+      package = pkgs.editorconfig-checker;
+      category = "general commands";
+      help = "Checker for editorconfig conformance";
+    }
+    {
+      package = pkgs.R;
+      category = "general commands";
+      help = "Statistical modelling tool";
+    }
+
+    {
+      package = cell.packages.fix-cabal-fmt;
+      category = "haskell";
+      help = "Format all cabal files in-place";
+    }
     {
       package = cell.packages.fix-stylish-haskell;
-      category = "plutus";
+      category = "haskell";
       help = "Run stylish-haskell on all haskell files in-place";
     }
+    {
+      package = cell.packages.cabal-install;
+      name = "cabal";
+      category = "haskell";
+      help = "Haskell build tool";
+    }
+    {
+      package = cell.packages.haskell-language-server;
+      name = "haskell-language-server";
+      category = "haskell";
+      help = "Haskell Language Server";
+    }
+    {
+      package = cell.packages.hlint;
+      name = "hlint";
+      category = "haskell";
+      help = "Haskell linting tool";
+    }
+    {
+      package = cell.packages.stylish-haskell;
+      name = "stylish-haskell";
+      category = "haskell";
+      help = "Haskell code formatter";
+    }
+    {
+      package = cell.packages.cabal-fmt;
+      name = "cabal-fmt";
+      category = "haskell";
+      help = "Cabal file formatter";
+    }
+
+    {
+      package = cell.packages.sphinx-toolchain;
+      name = "sphinx";
```

Comment:
Unfortunately this doesn't work:
```
[plutus-shell]$ sphinx
bash: sphinx: command not found
```
`cell.packages.sphinx-toolchain` is an ad-hoc python env with relevant sphinx packages included.
It should be something like this: 
```
  {
       package = cell.packages.sphinx-toolchain;
       name = "python";
       category = "docs";
       help = "Python interpreter with sphinx toolchain included";
     }
```

Reply:
Ah, it pulls in `sphinx-build` though. I guess what I wanted was to say "this provides you with `sphinx-build`" since otherwise you might not know that's there.

Reply:
I'll remove this for now.